### DISCUSSION TITLE
Dynamite Core Team Meeting 22nd March

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#373] [Announcements]
Hello @dynamics-of-stellar-systems/dynamite-maintainers,
group meeting has been shifted this week to 1:30pm, so the question is, if enough people are around to join for a Dynamite meeting at around 3pm. If not we start after the Easter holidays (Fri. 5th April) at 1pm. Please react to this announcement, if you want the meeting this week. Thanks!

**Comment by maindlt** [comment]
Hi all, for me it would be ok to dial in at 3pm (I'm back in CY) ;-) Cheers, Thomas

**Comment by sthater** [comment]
I would be available on the 22nd :) Either 1.30 or 3 pm :)

**Comment by azocchi** [comment]
I don't think that I will be able to make it tomorrow, sorry! 
I would be available on Friday afternoon (22.03), but if that does not work, I will be there on 05.04! ;)

**Comment by afeldmei** [comment]
Hello, 

I can’t come to a meeting today at 15:00. On Friday, 22.3., I can only make it after 14:00. 
I am available on 5.4.

Anja

**Comment by boeckera** [comment]
sorry guys I messed up I thought the group meeting is on Friday 1:30pm this week and not today (Wednesday). So I guess we stay at the usual 1pm on Friday.

**Comment by sthater** [comment]
Unfortunately, I cannot join tomorrow :( I have to go to the doctor because of my flu :(

**Comment by boeckera** [comment]
We use the normal dynamite zoom link
